tst/arguments_manager_test.cpp

```diff
@@ -0,0 +1,92 @@
+//
+// Created by kirill on 20.04.2020.
+//
+
+#include <gtest/gtest.h>
+#include "arguments_manager.h"
+#include "error_codes.h"
+#include <iostream>
+#include <color.h>
+#include <cstdlib>
+
+constexpr size_t arguments_buffer_size = 64;
+constexpr int default_unit_size = 1024*1024; // == 1024 KB == 1MB
+
+int argc;
```

Comment:
all declarations above would be better to put into fixture and then use created fixture (TEST(blabla, blabla) -> TEST_F(fixture_name, test_name)